osmaxx-py/excerptconverter/converter_helper.py

```diff
@@ -70,14 +72,12 @@ def inform_user(self, message_type, message_text, email=True):
 
         if email:
             if hasattr(self.user, 'email'):
-                pass
-                # FIXME: fix email sending
-                # send_mail(
-                #     '[OSMAXX] '+message_text,
-                #     message_text,
-                #     'no-reply@osmaxx.hsr.ch',
-                #     [self.user.email]
-                # )
+                mail.send_mail(
+                    '[OSMAXX] '+message_text,
```

Comment:
Isn't the hardcoded prefix here redundant to the `EMAIL_SUBJECT_PREFIX` environment variable?